Makefile

```diff
@@ -61,22 +117,20 @@ constraints.txt: dsl/requirements.txt fv3core/requirements.txt pace-util/require
 	sed -i.bak '/\@ git+https/d' constraints.txt
 	rm -f constraints.txt.bak
 
-physics_savepoint_tests:
-	$(MAKE) -C fv3gfs-physics $@
-
-physics_savepoint_tests_mpi:
-	$(MAKE) -C fv3gfs-physics $@
+physics_savepoint_tests: build
+	TARGET=physics $(MAKE) get_test_data
+	$(CONTAINER_CMD) $(CONTAINER_FLAGS) bash -c "pip3 list && cd $(PACE_PATH) && pytest --data_path=$(EXPERIMENT_DATA_RUN)/physics/ $(TEST_ARGS) $(PHYSICS_THRESH_ARGS) $(PACE_PATH)/fv3gfs-physics/tests/savepoint"
 
-update_submodules_venv:
-	if [ ! -f $(CWD)/external/daint_venv/install.sh  ]; then \
-                git submodule update --init external/daint_venv; \
-        fi
+physics_savepoint_tests_mpi: build
+	TARGET=physics $(MAKE) get_test_data
+	$(CONTAINER_CMD) $(CONTAINER_FLAGS) bash -c "pip3 list && cd $(PACE_PATH) && $(MPIRUN_CALL) python -m mpi4py -m pytest --maxfail=1 --data_path=$(EXPERIMENT_DATA_RUN)/physics/ $(TEST_ARGS) $(PHYSICS_THRESH_ARGS) -m parallel $(PACE_PATH)/fv3gfs-physics/tests/savepoint"
 
-test_driver:
-	DEV=$(DEV) $(MAKE) -C driver test
+test_main: build
+	$(CONTAINER_CMD) $(CONTAINER_FLAGS) bash -c "pip3 list && cd $(PACE_PATH) && pytest $(TEST_ARGS) $(PACE_PATH)/tests/main"
 
-driver_savepoint_tests_mpi:
-	DEV=$(DEV) $(MAKE) -C  fv3gfs-physics $@
+driver_savepoint_tests_mpi: build
+	TARGET=driver $(MAKE) get_test_data
```

Comment:
Driver test should also use the same dataset as physics

Reply:
I'm pretty sure the target is driver for these tests, based on [this deleted code](https://github.com/ai2cm/pace/pull/293/files#diff-e9d26bfeafc975a307dd2598ca9dde950de9eb9bf303977346b96cfa10076cfeL94). Can you look at it and confirm for me whether the target is driver or physics?